spanner/google/cloud/spanner/__init__.py

```diff
@@ -18,6 +18,13 @@
 import pkg_resources
 __version__ = pkg_resources.get_distribution('google-cloud-spanner').version
 
+from google.cloud.spanner._helpers import BOOL_PARAM_TYPE
```

Comment:
@tseaver do you think it's worthwhile for us to introduce a `types` module to hold these constants? (like we've done with gapic clients like vision, speech, etc.)